src/pages/iou/IOUModal.js

```diff
@@ -223,9 +227,39 @@ class IOUModal extends Component {
     }
 
     /**
+     * Update the payment type
+     *
+     * @param {String} paymentType
+     */
+    updatePaymentType(paymentType) {
+        this.setState({
+            paymentType,
+        });
+    }
+
+    /**
+     * Create the IOU transaction
+     *
      * @param {Array} [splits]
      */
-    createTransaction(splits) {
+    confirm(splits) {
+        if (this.state.iouType === 'send') {
+            payIOUReport({
+                chatReportID: this.props.route.params.reportID,
+                reportID: 0,
+                paymentMethodType: this.state.paymentType,
+
+                // should send in cents to API
+                amount: Math.round(this.state.amount * 100),
+                currency: this.props.iou.selectedCurrencyCode,
+
+                // submitterPayPalMeAddress: this.props.iouReport.submitterPayPalMeAddress,
+                // submitterPhoneNumber: this.submitterPhoneNumber,
```

Comment:
I think your comment might've gotten deleted, but that was a great question about what to do for these values. To be clear, we will need either one of them depending on the payment option the user selected (`paypal` or `venmo` respectively). So we will need an alternative way to get these if the user is trying to use them.

Reply:
Looks like the `paypal` address is just an NVP `NVP.PAYPALME_ADDRESS`. I'm thinking we might need to fetch this info when a user initially signs in and store it as an Onyx key-value.

Reply:
I am also working on this for the payment button and I'll cc you over there

Reply:
@nickmurray47 I am trying to figure out how to do this, I guess we will have to call the `nvp` as we get `personalDetails` for other account too, but will it not be too much inefficient performance wise?